### DISCUSSION TITLE
Where can I find the source code frontend of codesandbox.io only?

### DISCUSSION THREAD

**Discussion by ghost** [codesandbox/codesandbox-client#6874] [Q&A] [answered]
Hi all! I would like to know where I can find the source code of codesandbox.io(frontend)... I'm trying to look for that source code and I can't find it. Could someone help me? Where can I find the source code for the graphical interface of codesandbox.io only?

**Comment by sannek** [comment, marked as answer]
I'm not sure exactly what you mean, since  the codesandbox.io frontend code lives in this repository you're commenting on. If you're looking more specifically for the static pages, those live in [this subfolder](https://github.com/codesandbox/codesandbox-client/tree/master/packages/homepage).

If you mean something else, can you give a bit more detail?

**Reply by ghost** [reply]
Hi! So... this - It wouldn't be the homepage, it would be the IDE:
![image](https://user-images.githubusercontent.com/111701513/189316836-ae9e4aff-48e5-407d-9cf2-101324d24e1f.png)

**Reply by sannek** [reply]
It is all in this repository, but you might have to dig around a llittle bit. The best starting point is probably [here](https://github.com/codesandbox/codesandbox-client/tree/master/packages/app).
